### DISCUSSION TITLE
Param.json error in DPGEN_V10.6

### DISCUSSION THREAD

**Discussion by liu781319869** [deepmodeling/dpgen#868] [General]
**Summary**

When running, the train.log file in the dp stage will report the following error when it is in the dpgen0..10.6 version, but there is no such error under the same parameters as the previous dpgen0.10.5.
**DPGEN Version and Platform**
DPGEN_0.10.6  

Here is the training part in param.json!!!!!!
![image](https://user-images.githubusercontent.com/52098480/184308767-1f998145-78d8-4f02-9063-4db7ab8a6ea5.png)
And the error in train.log
![image](https://user-images.githubusercontent.com/52098480/184309238-93d5192e-8a3d-4052-a30b-88e611c550b1.png)

In the github.com/deepmodeling, the dpgen param.json is  in the following:
![b937b48bbc0018e33452891f3789ceb](https://user-images.githubusercontent.com/52098480/184310148-7bdead0d-fd79-4058-bb8b-b23e89e4d074.png)

**Job submission and computing cluster configuration**

If it's a bug, I think it's time to fix it and update the relevant documentation on Github in real time, or annotate the relevant parameter updates in different versions.
**Details**

**Comment by wanghan-iapcm** [comment]
It seems that you are using deepmd-kit v2.x and the training script is provided in the deepmd-kit v1.x format.
